saleor/settings.py

```diff
@@ -599,6 +599,10 @@ def get_host():
         "schedule": timedelta(seconds=20),
         "options": {"expires": BEAT_UPDATE_SEARCH_EXPIRE_AFTER_SEC},
     },
+    "expire-orders": {
+        "task": "saleor.order.tasks.expire_orders_task",
+        "schedule": crontab(minute="*/5"),
```

Comment:
Why not `timedelta(minutes=5),`? It should be more readable with `timedelta`

Reply:
Also, I would create a settings variable to control it. In case when the interval is too low or too big, we will be able to control it.